pkg/utils/random/random.go

```diff
@@ -1,3 +1,5 @@
+// random helps generate random strings & numbers
+// nolint
```

Comment:
If generating UUIDs is our only direct requirement, we can maybe offload it to https://github.com/google/uuid (BSD-3-clause) or https://github.com/satori/go.uuid (MIT license).